Bottom: 74c3500cf3e1f5cf1df8ac4100ab0efddc996e15
Top:    9e12ec9e014127ce37fdda36f87a03d33b4756a0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-14 01:06:40 +0100

Refresh of gtk-gl-refactor-gl-colour-setu

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b6f21e5..10ecbc2 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -358,8 +358,6 @@ set_gl_color_for_gc (hidGC gc)
   double r, g, b, a;
   a = 1.0;
 
-  current_gc = gc;
-
   if (old_name != NULL)
     {
       if (strcmp (gc->colorname, old_name) == 0)
@@ -455,8 +453,6 @@ set_gl_color_for_gc (hidGC gc)
 void
 ghid_set_color (hidGC gc, const char *name)
 {
-  render_priv *priv = gport->render_priv;
-
   current_gc = gc;
 
   gc->colorname = (char *) name;
@@ -518,7 +514,7 @@ use_gc (hidGC gc)
 
   current_gc = gc;
 
-  set_gl_colour_for_gc (gc);
+  set_gl_color_for_gc (gc);
   return 1;
 }
